docs/Containerfile.5.md

```diff
@@ -61,8 +61,10 @@ A Containerfile is similar to a Makefile.
   `FROM image@digest`
 
   -- The **FROM** instruction sets the base image for subsequent instructions. A
-  valid Containerfile must have **FROM** as its first instruction. The image can be any
-  valid image. It is easy to start by pulling an image from the public
+  valid Containerfile must have either **ARG** or *FROM** as its first instruction.
+  If **FROM** is not the first instruction in the file, it may only be preceded by
+  one or more ARG instructions, which declare arguments that are used in FROM lines in the Containerfile.
```

Comment:
Just to nitpick, aren't ARGs only used by the next FROM command.  IE 
ARG FOO=bar
from fedora
from $FOO

Does not work. From reading this description, I would assume it does.